api/helpers.c

```diff
@@ -331,7 +372,11 @@ void emit_fcall(mambo_context *ctx, void *function_ptr) {
   int ret = __emit_branch_cond(ctx->code.inst_type, ctx->code.write_p, (uintptr_t)function_ptr, AL, true);
   if (ret == 0) return;
 
-  emit_set_reg(ctx, lr, (uintptr_t)function_ptr);
+  #ifdef __riscv
```

Comment:
We'd be better off defining `lr` as an alias on RISC-V and `ra` on arm/aarch64 to help with plugin portability regardless of the naming conventions used.